kombu/transport/SQS.py

```diff
@@ -450,6 +454,29 @@ def visibility_timeout(self):
         return (self.transport_options.get('visibility_timeout') or
                 self.default_visibility_timeout)
 
+    @cached_property
+    def predefined_queue_urls(self):
+        """
+        Map of queue_name to queue URL. For example:
+        {"my-great-queue": "https://ap-southeast-2.queue.amazonaws.com/1234567890/my-great-queue"}
+
+        This is useful when you have been supplied an already existing SQS resource, but not
+        permissions to create your own, or list your own.
+
+        Often used with never_list_queues.
+        """
+        return self.transport_options.get('predefined_queue_urls', None)
+
+    @cached_property
+    def never_list_queues(self):
```

Comment:
I don't think that you need this setting. I believe it is preferrable to just skip the queue URL listing request, when the queue URL mapping has been given with `predefined_queue_urls`. Seems clearer from a user's point of view.